build.rs

```diff
@@ -20,8 +20,27 @@ fn main() {
     env::var("PROFILE").unwrap()
   };
 
+  // Equivalent to target arch != host arch
+  let is_different_target_arch =
+    env::var("CARGO_CFG_TARGET_ARCH").unwrap().as_str() != env::var("HOST")
+      .unwrap()
+      .as_str()
+      .split("-")
+      .collect::<Vec<&str>>()[0];
+
+  // If we are not using a standard target
+  let is_nonstandard_target =
```

Comment:
Can you list some example strings here that we might see for "HOST" and "TARGET" ? What is an example of a non-standard target? it's not immediately clear to me.

Reply:
`HOST`, `TARGET`, and `CARGO_CFG_TARGET_ARCH` are standard variables provided by cargo and documented here https://doc.rust-lang.org/cargo/reference/environment-variables.html
I might rename this `is_not_default_target`. The output of `rustup target list` will tell you what is default for your install of cargo/rustup.

Reply:
Actually I like `is_default_target` better.

Reply:
Sure - but some concrete examples are always very valuable when debugging these things - I guess host is "linux-x64" ? or is it "linux-misc-x64"? No one remembers and if it's not listed it will require me to println it later.

Reply:
I added some more detailed comments
Including how to check your default target: `rustup target list`

Reply:
It would just be nice to know what these values are concretely on your machine (since this will be their values in 90% of cross-compile cases)